package.json

```diff
@@ -77,6 +77,8 @@
     "test:e2e:watch": "vitest watch tests",
     "test:all": "vitest run",
     "test:watch": "vitest watch",
+    "test:examples": "vitest --config ./vitest.examples.config.ts run tests/e2e/examples",
+    "test:examples:watch": "vitest --config ./vitest.examples.config.ts run tests/e2e/examples",
```

Comment:
We must exclude `examples` from standalone `e2e`; otherwise, we would run them twice.

Reply:
Is it ok to move the `examples.test.ts` to another directory like `/tests/examples`?

Reply:
Yes